### ISSUE TITLE
Item missing !

### ISSUE THREAD

**Issue by Ruby721** [MistCore/MistCore#7]
There is a problem with item missing. When you add item, then relog. Item is missing. Really don't know how to fix. Lot's if people said that is the problem in Firewall or connectin. I Allowed all and still the same problem.

**Comment by Kittnz** [comment]
I think it's more of player saving issue or opcode. Unsure tho

**Comment by Dokman** [comment]
yes i think that is the code that doesn't save the equipment i think the opcode is solved i don't know now i'll watch the opcodes

**Comment by Ruby721** [comment]
Can you tell me where the opcodes are ?

**Comment by Ruby721** [comment]
Ok. Item missing bug is problem in transmog. When you transmog somethink and then add some item... next relog will miss.

**Comment by Dokman** [comment]
we have to watch the updated transmog of trintycore that works prefectly

**Comment by Ruby721** [comment]
So where is the problem ? I create a char add full set and etc, relog... OK... when i transmog somethink add new item.. then relog... Item missing.

**Comment by Kittnz** [comment]
You might wanna disable that for now until, however it seems that it doesn't save at relog or something along those lines

**Comment by Dokman** [comment]
yes

**Comment by Dokman** [comment]
i am busy for 1 or 2 months more, when i have time i'll try to fix it, but i don't know where to begin xD
